src/h3lib/include/linkedGeo.h

```diff
@@ -21,9 +21,36 @@
 #define LINKED_GEO_H
 
 #include <stdlib.h>
+#include "bbox.h"
 #include "geoCoord.h"
 #include "h3api.h"
 
+// Error codes for normalizeMultiPolygon
+#define NORMALIZATION_SUCCESS 0
+#define NORMALIZATION_ERR_MULTIPLE_POLYGONS 1
+#define NORMALIZATION_ERR_UNASSIGNED_HOLES 2
+
+/** Macro: Init iteration vars for LinkedGeoLoop */
+#define INIT_ITERATION_LINKED_LOOP       \
```

Comment:
Now that this is broken out, might be worth a comment explaining that these are meant for use with `polygonAlgos.h`